src/wallet/Web3WalletProvider.ts

```diff
@@ -97,15 +97,20 @@ export class Web3WalletProvider {
 		if (!walletType) throw new Error('Please provide a Wallet type to connect with.')
 
 		if (this[walletType as keyof Web3WalletProvider]) {
-			// @ts-ignore
-			// TODO: this address is null for the flow network
-			// Actual connected address is get in flowSubscribe function
-			const address = await this[walletType as keyof Web3WalletProvider](checkConnectionOnly)
-			logger(2, 'address', address)
+			try {
+				// @ts-ignore
+				// TODO: this address is null for the flow network
```

Comment:
I think this TODO is outdated after recent changes to the Flow integration. Can you check and remove if so?

Reply:
yes, outdated, removed.